packages/example-next-13-pages-and-app/README.md

```diff
@@ -0,0 +1,3 @@
+# example-next-13
```

Comment:
This file looks outdated.

Maybe you could explain the important parts here, e.g. that `i18n` must not be added to `next.config.js`?